lib/utils.ts

```diff
@@ -25,6 +26,15 @@ export interface IPeerRequest extends Request {
     realIp?: string;
 }
 
+export interface ISignalerEvents {
```

Comment:
does it makes sense to pull this into utils and export it? (is it reused?) if not, we should probably do as you've done with the `MyEmitter` type and inline it with no export in `peer-list.ts`

Reply:
I also thought that might be better. However, Andrei said he wanted all interfaces in one place (even non-reused) which is why I put it there. I'm thinking we either leave it in utils or take it out and also move the other one-use stuff (like IBuffer) to their respective files to maintain consistency.

Reply:
You can leave it for now, i guess. seems like refactoring all of em would be a fair bit of work.